playbooks/roles/streisand-gateway/tasks/main.yml

```diff
@@ -70,6 +70,13 @@
     - "{{ nginx_private_key }}"
     - "{{ streisand_gateway_password_file }}"
 
+- name: Copy the streisand gateway password file locally
```

Comment:
The same question of whether or not this should be Vagrant-specific and applied using a conditional would also apply here (as it did for the Shadowsocks password).